docs/topics/asset/giveaway/multiple.md

```diff
@@ -0,0 +1,84 @@
+# Asset Multiple Giveaway Smart Contract
+
+## Overview
+
+Admin locks multiple asset with merkle roots and user can claim the asset with matching params.
+
+Similar to Land presales and the Asset giveaway, we have an off-chain merkle tree. The key is the user’s Ethereum address. This is a reusable contract, so multiple off-chain merkle trees can be used. Once a user claims their tokens relating to a particular merkle tree, it cannot claim them again.
+
+On the frontend, we can show the list of tokens the user is entitled to, for each merkle tree. Once the user clicks ‘claim’, the frontend can fetch from the backend the relevant proof(s) and any extra parameter to perform the tx call.
+
+## Features
+
+### 1. Add New Giveaway
+
+Only admin can add merkle roots and expiry times. Admin is determined while deploying the contract.
+
+### 2. Get Claimed Status
+
+To check the merkle trees for a particular user.
+
+### 3. Claim MultipleTokens From Multiple MerkleTree
+
+Submit multiple Claims with multiple proofs and multiple merkle root hashes
+For multiple tokens from multiple giveaways (i.e. a Claim array) - see below for Claim object
```

Comment:
Submit multiple Claims with multiple proofs and multiple merkle root hashes.
For.....